docs/dam/integration_with_dxp.md

```diff
@@ -0,0 +1,226 @@
+---
+description: Integrate DAM with your Ibexa DXP project to use an effective tool for managing marketing assets.
+---
+
+
+# Integrate DAM with DXP
+
+DAM integration with [[= product_name =]] is dedicated to [[= product_name_base =]] users who want to use [[= product_name_base =]] as a data storage. It enables using assets repository stored in the DXP in the platform to build and insert marketing assets into campaign emails.
```

Comment:
This sentence "to build and insert marketing assets into campaign emails." still "smells" of Actito integration.